app/views/user_panel/profile/index.html.slim

```diff
@@ -1,8 +1,33 @@
-h1.text-white User panel
+h1 User panel
 
-.card.w-75
-  .card-body
-    h5.card-title Edit your data
-    p.card-text You can change password, email and cancel your account
-    a.btn.btn-primary href=edit_user_registration_path  Edit data
+.row
+  .card.w-75
+    .card-body
+      h5.card-title Edit your data
+      p.card-text You can change password, email and cancel your account
```

Comment:
W `cancel` chodzi o całkowite usunięcie konta z bazy danych? Jeśli tak, to chyba lepiej użyć `delete`
```suggestion
      p.card-text You can change password, email or delete your account
```